js/load.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+(() => {
+  const URL = 'https://21.javascript.pages.academy/keksobooking/data';
+  const TIMEOUT_IN_MS = 2000;
```

Comment:
Поставь побольше, за 2 секунды ничего не будет подгружаться в какой-то части случаев
```suggestion
  const TIMEOUT_IN_MS = 10000;
```